libobs-opengl/gl-wayland-egl.c

```diff
@@ -383,6 +383,17 @@ static bool gl_wayland_egl_device_query_dmabuf_modifiers_for_format(
 		plat->display, drm_format, modifiers, n_modifiers);
 }
 
+static struct gs_texture *gl_wayland_egl_device_texture_create_from_pixmap(
+	gs_device_t *device, uint32_t width, uint32_t height,
+	enum gs_color_format color_format, uint32_t target, void *pixmap)
+{
+	struct gl_platform *plat = device->plat;
+
+	return gl_egl_create_texture_from_pixmap(plat->display, width, height,
+						 color_format, target,
+						 (EGLClientBuffer)pixmap);
```

Comment:
Does it make sense to implement this vfunc on Wayland / EGL? I'm about 78% sure EGL will crash.

Reply:
It shouldnt crash, it should return a null texture because eglCreateImage will fail with BADMATCH since EGL_NATIVE_PIXMAP_KHR shouldnt work on wayland and return EGL_NO_IMAGE (but i havnt tested it). null texture is expected when we fail to import and is used for e.g. pipewire fallback to shm so it should be handled by all callers.

I could just return NULL here to make it more clear.